account_credit_control/models/account_invoice.py

```diff
@@ -18,7 +18,8 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from openerp import models, fields, api, _
+from odoo import models, fields, api, _
```

Comment:
Indeed, but it's a nice to have if you don't mind doing the rest of the comments. If not, as said, don't block the PR for that.